### PR TITLE
Feat(global): bootstraping refactoring

### PR DESCRIPTION
**Pull request by maksk7**

### CODE REVIEW

File: contracts/lockdrop/src/contract.rs [hunk 1]
```diff
@@ -582,8 +722,10 @@ pub fn handle_initialize_pool(
         let res: Option<cw20::MinterResponse> = deps
             .querier
             .query_wasm_smart(&terraswap_lp_token, &Cw20QueryMsg::Minter {})?;
-        deps.api
-            .addr_validate(&res.expect("No minter for the LP token!").minter)?
+        deps.api.addr_validate(
```
Comment (epanchee) [inline contracts/lockdrop/src/contract.rs:725]: Better to use `addr_validate_to_lower`

File: contracts/airdrop/src/contract.rs [hunk 2]
```diff
@@ -98,16 +137,27 @@ pub fn execute(
...
+/// * **deps** is an object of type [`DepsMut`].
+///
+/// * **env** is an object of type [`Env`].
+///
+/// * **info** is an object of type [`MessageInfo`].
+///
+/// * **cw20_msg** is an object of type [`Cw20ReceiveMsg`]. This is the CW20 message that has to be processed.
 pub fn receive_cw20(
     deps: DepsMut,
     _env: Env,
```
[the remark this answers is not in the document]
Comment (maksk7) [reply contracts/airdrop/src/contract.rs]: Done, thanks

File: contracts/airdrop/src/contract.rs [hunk 3]
```diff
@@ -132,27 +182,58 @@ pub fn receive_cw20(
...
-// Handle functions
-//----------------------------------------------------------------------------------------
-
-/// @dev Admin function to update Configuration parameters
-/// @param new_config : Same as InstantiateMsg struct
+/// Admin function to update any of the configuration parameters.. Returns a [`StdError`] on failure.
+/// ## Params
+/// * **deps** is an object of type [`DepsMut`].
+///
+/// * **_env** is an object of type [`Env`].
```
[the remark this answers is not in the document]
Comment (maksk7) [reply contracts/airdrop/src/contract.rs]: Done

File: contracts/lockdrop/src/contract.rs (hunk 1 above)
Comment (maksk7) [reply contracts/lockdrop/src/contract.rs]: Done, thanks

**Review by P-Yevhenii — APPROVED** [review]

**Review by epanchee — APPROVED** [review]

**Pull request merged**